eva_cttv_pipeline/clinvar_to_evidence_strings.py

```diff
@@ -7,13 +7,23 @@
 import jsonschema
 import xlrd
 
-from eva_cttv_pipeline import cellbase_records, efo_term, clinvar_record, consequence_type, config, evidence_strings
+from eva_cttv_pipeline import cellbase_records, efo_term, consequence_type, config, \
+    evidence_strings
+from eva_cttv_pipeline import clinvar_record as clinvar_record_module
```

Comment:
Maybe the module could be renamed simply to `clinvar`. Are there any classes directly related to ClinVar apart from `ClinvarRecord`?